OrbitGl/DataManager.h

```diff
@@ -36,11 +36,15 @@ class DataManager final {
   void set_selected_text_box(const TextBox* text_box);
   void set_selected_process(std::shared_ptr<Process> process);
 
+  void SelectTracepoint(std::string tracepoint_name);
+  void DeselectTracepoint(std::string tracepoint_name);
```

Comment:
These functions are not called. - Also should these record categories as well? Maybe store TracepointInfo proto instead of just name?